instances/technique/cellAttachedPatchClamp.jsonld

```diff
@@ -4,11 +4,11 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/cellAttachedPatchClamp",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": null,
-  "description": null,
+  "definition": "Cell-attached patch clamp is a patch clamp recording technique used in electrophysiology in which the intracellular integrity of the cell is preserved.",
```

Comment:
I would typically not reuse the definition again in the description to avoid redundancy for the reader. 

I also think the definition is not fully correct or at least misleading. Cell-attached means not only that the intracellular integrity is preserved but also the cell membrane remains intact.

Maybe rephrase definition to: 
"'Cell-attached patch clamp' is an intracellular electrophysiology technique that fully preserves the intracellular integrity by forming a megaohm or gigaohm seal, leaving the cell membrane intact."

I would extract the description from this paper (with ref at the end) https://www.sciencedirect.com/science/article/pii/B9780080552323600121

Reply:
Anything in particular? I am not sure what you are looking for